uwsgi.ini

```diff
@@ -0,0 +1,12 @@
+[uwsgi]
+http = :5000
+module = application:application
+buffer-size = 32768
```

Comment:
Was 32k a guesstimate or do we know our max request size?

Reply:
Our payloads vary between 6-10k, so it was a guesstimate based on that with headroom.